corehq/apps/fixtures/utils.py

```diff
@@ -55,6 +57,10 @@ def get_index_schema_node(fixture_id, attrs_to_index):
     return node
 
 
-def clear_fixture_cache(domain):
+def clear_fixture_cache(domain, data_type_ids):
+    if isinstance(data_type_ids, str) or not hasattr(data_type_ids, '__iter__'):
+        data_type_ids = [data_type_ids]
```

Comment:
Always pass in a sequence of ids rather than doing type checking here.
```suggestion
    assert not isinstance(data_type_ids, str), 'expected list or set, got str'
```

Reply:
[51dfb13](https://github.com/dimagi/commcare-hq/pull/36091/commits/51dfb13ab90bb440308f2bde6bf489d515355181)